deploy/vagrant/ufs/hadoop1/init.sh

```diff
@@ -6,9 +6,6 @@ NODES=`cat /tachyon/conf/workers`
 
 cd /vagrant/shared
 
-mkdir -p /tmp/hdfs-datanode
-mkdir -p /tmp/hadoop-tmpstore
```

Comment:
hadoop format will create these necessary paths. During my test, when these paths exist, datanode won't be started, encountering the same problem here: http://stackoverflow.com/questions/10097246/no-data-nodes-are-started